pkg/datastoresrp/api/v20231001preview/mongodatabase_conversion_test.go

```diff
@@ -294,9 +294,7 @@ func TestMongoDatabase_ConvertDataModelToVersioned(t *testing.T) {
 					ProvisioningState:    to.Ptr(ProvisioningStateAccepted),
 					Recipe:               &Recipe{Name: to.Ptr("cosmosdb"), Parameters: map[string]interface{}{"foo": "bar"}},
 					Username:             to.Ptr(""),
-					Status: &ResourceStatus{
-						OutputResources: nil,
-					},
+					Status:               resourcetypeutil.MustPopulateResourceStatusWithRecipe(&ResourceStatus{}),
```

Comment:
Cannot use MustPopulateResourceStatusWithRecipe due to OutputResources: nil check

Reply:
Not sure why the output resource was nil, but looks like we were able to replace it with MustPopulateResourceStatusWithRecipe, so resolving this.